### PR TITLE
Misc UI fixes for configs and prebuilds

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

Fixes some small issues with prebuilds and configs:
- in lists, fix layouts where the configuration name is super long 
- fixed prebuild status icons to be consistent and similar to the rest

## Related Issue(s)

Fixes #

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - ft-repo-pr16276b329d</li>
	<li><b>🔗 URL</b> - <a href="https://ft-repo-pr16276b329d.preview.gitpod-dev.com/workspaces" target="_blank">ft-repo-pr16276b329d.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - ft-repo-prebuilds-fixes-gha.22946</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-ft-repo-pr16276b329d%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-core-dev" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx [hunk 1]
```diff
@@ -158,11 +158,11 @@ export const PrebuildDetailPage: FC = () => {
                                             </div>
                                         )}
                                     </div>
-                                    <div className="flex gap-1">
+                                    <div className="flex gap-1 items-center">
```
Comment (filiptronicek) [inline components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx:161]: fixes misaligned author info and unhelpful alt text

File: components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx [hunk 2]
```diff
@@ -158,11 +158,11 @@ export const PrebuildDetailPage: FC = () => {
...
                                         <img
                                             className="w-5 h-5 rounded-full"
                                             src={info.prebuild.commit?.author?.avatarUrl}
-                                            alt={info.prebuild.commit?.author?.name}
+                                            alt=""
```
Comment (Siddhant-K-code) [inline components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx:165]: didn't understand the intent for this change 🤔 

was it returning the wrong data? If yes, we can still keep it a generic one
```suggestion
                                            alt="User Avatar"
```

**Review by Siddhant-K-code — APPROVED** [review]
LGTM (with a [small comment](https://github.com/gitpod-io/gitpod/pull/19419/files#r1489355901)) ✅

File: components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx (hunk 2 above)
Comment (filiptronicek) [reply components/dashboard/src/repositories/detail/prebuilds/PrebuildDetailPage.tsx:165]: It was not returning anything incorrect, but in this case, the alt text does not add value to screen readers and can be considered a decorative image. If we keep as-is, we're essentially repeating ourselves

- https://web.dev/shows/http-203/flf2vS0IoRs/
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#value

**Pull request merged**